TestCases/tutorials.py

```diff
@@ -213,10 +213,22 @@ def main():
     tutorial_design_multiobj.no_restart = True
     test_list.append(tutorial_design_multiobj)
 
+    # Multi Objective Design
+    testcase_su2_nastran            = TestCase('py_su2_nastran')
+    testcase_su2_nastran.cfg_dir    = "TestCases/py_su2_nastran"
```

Comment:
This file is supposed to get the grids from the tutorials repo, not from testcases.